.github/scripts/generate_ci_workflows.py

```diff
@@ -73,6 +73,9 @@ def __post_init__(self) -> None:
         if not self.enabled:
             self.reset_root_job()
             return
+
+        # Add ciflow/all to labels
+        self.labels.add('ciflow/all')
```

Comment:
My favorite remark: please replace string global variable (so that linter can catch typos in those)
```suggestion
        self.labels.add(CIFLOW_ALL)
```

Reply:
ok, for consistency I will change all of these into global constants, changes coming up